packages/Algernon.package/ALGClassType.class/instance/matchesComment..st

```diff
@@ -0,0 +1,3 @@
+accessing
+matchesComment: aString
```

Comment:
I'm not sure we need this method anymore. Could you check?

Reply:
We don't; I removed it.